controllers/testutils_test.go

```diff
@@ -1,15 +1,17 @@
-package controllers
+package controllers_test
```

Comment:
I like the package changes; however, the name of the file isn't great. Maybe `utils_test`?